### PR TITLE
add Unit Test #582

### PR DESCRIPTION
**Pull request by btyanagawamg**
Please review #582 .

A Unit Test to implementation for #582 is added .

### CODE REVIEW

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+            assertThat(form.getField("customerCode"), is("00000001"));
+            assertThat(form.getField("reserveNo"), is("00000001"));
+            assertThat(form.getField("remarks"), is("特になし"));
+            assertThat(form.getField("accomTel"), is("018-123-4567"));
+            assertThat(form.getField("customerPost"), is("276-0022"));
+            assertThat(form.getField("printDay"), is("2019/03/06"));
+            assertThat(form.getField("adultPrice"), is("375000"));
+            assertThat(form.getField("childPrice"), is("112500"));
+            assertThat(form.getField("sumPrice"), is("487500"));
+            assertThat(form.getField("paymentTimeLimit"), is("2019/03/24"));
```
Comment (yoshikawaa) [inline terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: - [ ] You should check whether or not unnecessary fields are registered.

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+        try {
+            reservationReportPdfStamperView.prepareResponse(request, response);
+            assertThat(response.getCharacterEncoding(), is(
+                    StandardCharsets.UTF_8.name()));
+        } catch (Exception e) {
+            e.printStackTrace();
+            fail(); // FAIL when exception is thrown
+        }
+    }
+}
```
Comment (yoshikawaa) [inline terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: - [ ] You should import class `com.lowagie.text.pdf.PdfStamper`.

https://github.com/terasolunaorg/terasoluna-tourreservation/blob/164a01ef7936041b8abe7f287f308a0c633f422f/terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperView.java#L52

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+            downloadPDFOutput.setCustomerBirth(sdf.parse("1975/01/05"));
+            downloadPDFOutput.setPrintDay(sdf.parse("2019/03/06"));
+        } catch (ParseException e) {
+            e.printStackTrace();
+            fail(); // FAIL when exception is thrown
+        }
+
+        model = new HashMap<String, Object>();
+        model.put("downloadPDFOutput", downloadPDFOutput);
+        model.put("downloadPDFName", "reservationReport");
```
Comment (yoshikawaa) [inline terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: - [ ] Why setup `model` at `@Before`? `model` is used only `testMergePdfDocument`.

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java (hunk 2 above)
Comment (yoshikawaa) [inline terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: - [ ] You should remove unnecessary `String.valueOf()`. The return value of `DateFormat#format(Date)` is String.

https://github.com/terasolunaorg/terasoluna-tourreservation/blob/164a01ef7936041b8abe7f287f308a0c633f422f/terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperView.java#L80-L81

Comment (yoshikawaa) [inline terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: - [ ] You should apply naming rule of constant. (`fieldName` to `FIELD_NAME`)

https://github.com/terasolunaorg/terasoluna-tourreservation/blob/164a01ef7936041b8abe7f287f308a0c633f422f/terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperView.java#L36-L40

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+        downloadPDFOutput.setPaymentTimeLimit("2019/03/24");
+        try {
+            downloadPDFOutput.setReservedDay(sdf.parse("2019/02/21"));
+            downloadPDFOutput.setDepDay(sdf.parse("2019/03/31"));
+            downloadPDFOutput.setCustomerBirth(sdf.parse("1975/01/05"));
+            downloadPDFOutput.setPrintDay(sdf.parse("2019/03/06"));
+        } catch (ParseException e) {
+            e.printStackTrace();
+            fail(); // FAIL when exception is thrown
+        }
```
[the remark this answers is not in the document]
Comment (btyanagawamg) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: 85c894352b6c54b1d80bd966700548f86b916597

I fixed policy in above commit .

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+
+    @Test
+    public void testGetUrl() {
+        try {
+            String url = reservationReportPdfStamperView.getUrl();
+            assertThat(url, is("classpath:reports/reservationReport.pdf"));
+        } catch (Exception e) {
+            e.printStackTrace();
+            fail(); // FAIL when exception is thrown
+        }
```
[the remark this answers is not in the document]
Comment (btyanagawamg) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: 85c894352b6c54b1d80bd966700548f86b916597

I fixed policy in above commit .

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+
+        request = new MockHttpServletRequest();
+        response = new MockHttpServletResponse();
+
+        ReflectionTestUtils.setField(reservationReportPdfStamperView,
+                "reservationReportPdfUrl",
+                "classpath:reports/reservationReport.pdf");
```
[the remark this answers is not in the document]
Comment (btyanagawamg) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: 02f509c30ab7811610f146badf91b91e988acd95

I fixed in above commit .

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java [hunk 7]
```diff
@@ -0,0 +1,207 @@
...
+            e.printStackTrace();
+            fail(); // FAIL when exception is thrown
+        }
+
+        model = new HashMap<String, Object>();
+        model.put("downloadPDFOutput", downloadPDFOutput);
+        model.put("downloadPDFName", "reservationReport");
+
+        try {
+            stamper = new PdfStamper(new PdfReader("src/main/resources/reports/reservationReport.pdf"), new ByteArrayOutputStream(OUTPUT_BYTE_ARRAY_INITIAL_SIZE));
```
[the remark this answers is not in the document]
Comment (btyanagawamg) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: 18e74c706589d5f89deddc2dc17db22677e4eebc

I fixed in above commit .

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java [hunk 8]
```diff
@@ -0,0 +1,207 @@
...
+            assertThat(form.getField("reserveNo"), is("00000001"));
+            assertThat(form.getField("remarks"), is("特になし"));
+            assertThat(form.getField("accomTel"), is("018-123-4567"));
+            assertThat(form.getField("customerPost"), is("276-0022"));
+            assertThat(form.getField("printDay"), is("2019/03/06"));
+            assertThat(form.getField("adultPrice"), is("375000"));
+            assertThat(form.getField("childPrice"), is("112500"));
+            assertThat(form.getField("sumPrice"), is("487500"));
+            assertThat(form.getField("paymentTimeLimit"), is("2019/03/24"));
+
```
[the remark this answers is not in the document]
Comment (btyanagawamg) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: `FormFlattening` and `FreeTextFlattening` properties will set in [PdfStamperImp](https://github.com/LibrePDF/OpenPDF/blob/master/openpdf/src/main/java/com/lowagie/text/pdf/PdfStamperImp.java#L85-L86) and those properties never be able to be accesssed outside, so those tests are not able to done .

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java [hunk 9]
```diff
@@ -0,0 +1,207 @@
...
+        } catch (DocumentException e) {
+            e.printStackTrace();
+            fail(); // FAIL when exception is thrown
+        } catch (IOException e) {
+            e.printStackTrace();
+            fail(); // FAIL when exception is thrown
+        }
+
+        request = new MockHttpServletRequest();
+        response = new MockHttpServletResponse();
```
[the remark this answers is not in the document]
Comment (btyanagawamg) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: 1e2915526f08c994877d6a97211bf657cf0ae0ad

I fixed in above commit .

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java [hunk 10]
```diff
@@ -0,0 +1,207 @@
...
+
+    @Test
+    public void testMergePdfDocument() {
+        try {
+            reservationReportPdfStamperView.mergePdfDocument(model, stamper,
+                    request, response);
+
+            AcroFields form = stamper.getAcroFields();
+            assertThat(form.getField("referenceName"), is(
+                    "TERASOLUNA TRAVEL CUSTOMER CENTER"));
```
[the remark this answers is not in the document]
Comment (btyanagawamg) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: 9368e45921afe04c0b2b2ea2d85d8019a9b5d85c

I fixed in above commit .

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java [hunk 11]
```diff
@@ -0,0 +1,207 @@
...
+    public void testPrepareResponse() {
+        try {
+            reservationReportPdfStamperView.prepareResponse(request, response);
+            assertThat(response.getCharacterEncoding(), is(
+                    StandardCharsets.UTF_8.name()));
+        } catch (Exception e) {
+            e.printStackTrace();
+            fail(); // FAIL when exception is thrown
+        }
+    }
```
[the remark this answers is not in the document]
Comment (btyanagawamg) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: b72fefe8ddbc9a98baf783a2fba8d921eff2dda2

I fixed in above commit .

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java (hunk 1 above)
Comment (btyanagawamg) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: fd313eba7b0eac1ceb70eee785765dd3b68fb95b

I fixed in above commit .

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java (hunk 2 above)
Comment (btyanagawamg) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: 131e3dec86748ee4026b19192fc9a06b14bd0ee8

I fixed in above commit .

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java (hunk 3 above)
Comment (btyanagawamg) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: b7c739f6c7a55250b7a1b9951b5ba091b399b99d

I fixed in above commit .

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java (hunk 2 above)
Comment (btyanagawamg) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: 577ecc86e68c72f77d52b39f5a12831546d9357b

I fixed in above commit .

Comment (btyanagawamg) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: e83dcda1e1056ae07e8b317eacc8fbf898b0bdb3

I fixed in above commit .

Comment (yoshikawaa) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: `SimpleDateFormat` is actually not constant. Better to reset only this.

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java (hunk 6 above)
[the remark this answers is not in the document]
Comment (yoshikawaa) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: It seems `reservationReportPdfUrl` is only required for `testGetUrl`.

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java (hunk 9 above)
[the remark this answers is not in the document]
Comment (yoshikawaa) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: > `PdfStamper stamper = null;`

Why initialize as `null`?
It seems enough as `= new PdfStamper(...);`.

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java (hunk 5 above)
[the remark this answers is not in the document]
Comment (yoshikawaa) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: Please explain your (or whole unit test) policy. I assumed `throws Exception`.

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java [hunk 12]
```diff
@@ -0,0 +1,203 @@
...
+        try {
+            form.decodeGenericDictionary(referenceNameMerged,
+                    referenceNameTextField);
+        } catch (IOException e) {
+            e.printStackTrace();
+            fail(); // FAIL when exception is thrown
+        } catch (DocumentException e) {
+            e.printStackTrace();
+            fail(); // FAIL when exception is thrown
+        }
```
Comment (yoshikawaa) [inline terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: - [ ] You can unit catch statements to catch super class (Exception) or multiple catch.

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java (hunk 9 above)
[the remark this answers is not in the document]
Comment (btyanagawamg) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: 9552c7f25bf93d8b63b8071016eb0f611c3a6da4

No, initialize not need.
I removed in above commit .

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java [hunk 13]
```diff
@@ -0,0 +1,203 @@
...
+
+        AcroFields form = stamper.getAcroFields();
+        assertThat(form.getField("referenceName"), is(
+                "TERASOLUNA TRAVEL CUSTOMER CENTER"));
+        AcroFields.Item referenceNameItem = form.getFieldItem("referenceName");
+        PdfDictionary referenceNameMerged = referenceNameItem.getMerged(0);
+        TextField referenceNameTextField = new TextField(null, null, null);
+        try {
+            form.decodeGenericDictionary(referenceNameMerged,
+                    referenceNameTextField);
```
Comment (yoshikawaa) [inline terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: - [ ] You should add comment to explain why necessary these codes.
(ex.. Can't get field properties directly from `AcroFields`)

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java [hunk 14]
```diff
@@ -0,0 +1,203 @@
...
+        } catch (IOException e) {
+            e.printStackTrace();
+            fail(); // FAIL when exception is thrown
+        } catch (DocumentException e) {
+            e.printStackTrace();
+            fail(); // FAIL when exception is thrown
+        }
+        assertThat(referenceNameTextField.getFontSize(), is(
+                REFERENCE_NAME_VARIABLE_FONTSIZE));
```
Comment (yoshikawaa) [inline terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: - [ ] You should assert pattern to change or not font size.

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java (hunk 8 above)
[the remark this answers is not in the document]
Comment (yoshikawaa) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: I recommend to use `ReflectionTestUtils` or `ReflectionUtils` for getting these properties.

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java (hunk 11 above)
[the remark this answers is not in the document]
Comment (yoshikawaa) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: Mockito enable to verify super class method.
https://blog.jdriven.com/2013/05/mock-superclass-method-with-mockito/
But I do not request this.

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java [hunk 15]
```diff
@@ -0,0 +1,203 @@
...
+        HttpServletRequest request = new MockHttpServletRequest();
+
+        HttpServletResponse response = new MockHttpServletResponse();
+
+        reservationReportPdfStamperView.prepareResponse(request, response);
+        assertThat(response.getHeader("Pragma"), is("private"));
+        assertThat(response.getHeader("Cache-Control"), is(
+                "private, must-revalidate"));
+        assertThat(response.getCharacterEncoding(), is(StandardCharsets.UTF_8
+                .name()));
```
Comment (yoshikawaa) [inline terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: I recmmend to add comment to explain `CharacterEncoding` is responsibility of this class , and Cache-Control Headers are responsibility of super classes.

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java (hunk 1 above)
Comment (yoshikawaa) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: If you assert size of fields, I recommend to do it first.

Comment (btyanagawamg) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: 2c950321d95a99d81e6f6318bce3d7095bee7d5c

I fixed in above commit .

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java (hunk 12 above)
Comment (btyanagawamg) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: 8e72b2ada7ab00f10862e82068a9fa6e42f80a6a

I changed to throw `Exception` in this place .

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java (hunk 8 above)
[the remark this answers is not in the document]
Comment (btyanagawamg) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: c2959e6a6c05dbbaa70073f9c24b5eed53c62034

I fixed in above commit .

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java (hunk 14 above)
Comment (btyanagawamg) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: fe28e636539e05b78e19e595568d92f2105c7b20

I add testcase in above commit .

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java (hunk 5 above)
[the remark this answers is not in the document]
Comment (btyanagawamg) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: The place to call the method of the class to be tested is enclosed by try-catch, and the other places are to `throws Exception` .

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java (hunk 6 above)
[the remark this answers is not in the document]
Comment (btyanagawamg) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: ee39b6734d15b70ad3e0c36e7560722b9062916c

I fixed in above commit .

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java (hunk 13 above)
Comment (btyanagawamg) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: ff4f82c5e6c519d108d0401bef87b9de5cdce191

I add comment in above commit .

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java (hunk 15 above)
Comment (btyanagawamg) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: 54e971965e11d5a723d864d7a1b7cd72631da35b

I add comment in above commit .

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java (hunk 8 above)
[the remark this answers is not in the document]
Comment (yoshikawaa) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: You should adjust to use `assertThat`.

```
assertThat((boolean) ReflectionTestUtils.getField(pdfStamperImp, "flat"), is(true));

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java (hunk 14 above)
Comment (yoshikawaa) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: In general, you should test with patterns that do not exceed the boundary value (30 Characters).

File: terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java (hunk 8 above)
[the remark this answers is not in the document]
Comment (btyanagawamg) [reply terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java]: 155b5e15314db2229e094eb5404afddbce708248

I changed `assertTrue` -> `assertThat` in above commit .
